developer-docs/README.md

```diff
@@ -48,17 +49,18 @@ Once you have pushed it to PyPi, add the git tag
 ```bash
 $ export RELEASE_VERSION=v$(python3 ./sematic/versions.py)
 $ git tag $RELEASE_VERSION
-$ git push --tags
+$ git push origin $RELEASE_VERSION
 ```
 
 Next, build and push the server image. Use the dockerfile at
-`docker/Dockerfile.server`. Use the wheel you built before in the directory
+`docker/Dockerfile.server`. Copy the wheel you built before in the directory
 you run.
 ```bash
 $ cd docker
-$ RELEASE_VERSION=v$(python3 ../sematic/versions.py)
+$ export RELEASE_VERSION=v$(python3 ../sematic/versions.py)
 $ docker build -t "sematicai/sematic-server:$RELEASE_VERSION" -f Dockerfile.server .
 $ docker push "sematicai/sematic-server:$RELEASE_VERSION"
```

Comment:
We have `make` command for this.
```
$ TAG=my_tag make release-server
```
(Requires the wheel to be in the `docker/` directory.

Reply:
Done.